rholang/src/main/scala/rholang/interpreter/Normalize.scala

```diff
@@ -22,18 +22,58 @@ trait GroundNormalizeVisitor extends AbsynGround.Visitor[Ground, Any]
   override def visit( gi: GroundUri, n: Any ) : Ground = GUri(gi.uri_)
 }
 
+trait NameNormalizeVisitor extends Name.Visitor[NameVisitOutputs, NameVisitInputs] {
+  val procVisitor : Proc.Visitor[ProcVisitOutputs, ProcVisitInputs]
```

Comment:
It is customary to place the colon for a type annotation directly after the identifier name (or parentheses, in the case of a function definition).  Should be:
```scala
val procVisitor: Proc.Visitor[ProcVisitOutputs, ProcVisitInputs]
```